### DISCUSSION TITLE
slow reload

### DISCUSSION THREAD

**Discussion by bottleoffire** [ExitedState/Stack-CPU-Simulator#7] [Q&A] [answered]
Why is there a slight lag when accessing a website for the first time?

**Comment by ExitedState** [comment, marked as answer]
I think it problem at JavaScript Execution. (have many MemoryOptions to generate a first time)
If I have time i will resolve it, thanks you

**Comment by bottleoffire** [comment]
ok krubb

**Comment by ExitedState** [comment]
I just updated the code. I resolved this problem by adding loading screen. (for ux reason)
